diff --git a/src/Element/Tablefield.php b/src/Element/Tablefield.php
index c2194c8..b31b21b 100644
--- a/src/Element/Tablefield.php
+++ b/src/Element/Tablefield.php
@@ -225,18 +225,6 @@ class Tablefield extends FormElement {
       ];
     }
 
-    $element['tablefield']['row_header'] = [
-      '#type' => 'checkbox',
-      '#title' => t('Display first row as a table header'),
-      '#default_value' => $row_header,
-    ];
-
-    $element['tablefield']['column_header'] = [
-      '#type' => 'checkbox',
-      '#title' => t('Display first column as a table header'),
-      '#default_value' => $column_header,
-    ];
-
     return $element;
   }
 
diff --git a/src/Plugin/Field/FieldWidget/TablefieldWidget.php b/src/Plugin/Field/FieldWidget/TablefieldWidget.php
index e37a1d5..135b263 100644
--- a/src/Plugin/Field/FieldWidget/TablefieldWidget.php
+++ b/src/Plugin/Field/FieldWidget/TablefieldWidget.php
@@ -153,6 +153,18 @@ class TablefieldWidget extends WidgetBase implements ContainerFactoryPluginInter
       '#description' => $this->t('This brief caption will be associated with the table and will help screen reader better describe the content within.'),
     ];
 
+    $element['row_header'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Display first row as a table header'),
+      '#default_value' => $row_header,
+    ];
+
+    $element['column_header'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Display first column as a table header'),
+      '#default_value' => $column_header,
+    ];
+
     $element = [
       '#type' => 'tablefield',
       '#input_type' => $this->getSetting('input_type'),
